knex/seeds/splash-pads.js

```diff
@@ -0,0 +1,21 @@
+
+exports.seed = async (knex) => {
+
+  const parks = [{ name: 'bartholomew', latitude: 30.305177, longitude: -97.697469 },
+  { name: 'chestnut', latitude: 30.277067251010195, longitude: -97.71699243984315 },
+  { name: 'eastwoods', latitude: 30.290943, longitude: -97.731715 },
+  { name: 'liz-carpenter', latitude: 30.26211031582992, longitude: -97.75441430399374 },
+  { name: 'lott', latitude: 30.271041206071896, longitude: -97.7297671152558 },
+  { name: 'metz', latitude: 30.252258120681006, longitude: -97.71810034083994 },
+  { name: 'mueller-branch-park', latitude: 30.300190, longitude: -97.702812 },
+  { name: 'ricky-guerrero', latitude: 30.247392178325605, longitude: -97.76433234845857 },
+  { name: 'rosewood', latitude: 30.27109617907063, longitude: -97.71390894361079 },
+  { name: 'pease', latitude: 30.281754288606752, longitude: -97.7521610530982 }]
+
+  await Promise.all(parks.map(async (park) => {
+    const parkExists = await knex.select('*').from('parks').where({ name: park.name })
+    if (parkExists.length === 0) {
+      knex('parks').insert(park);
```

Comment:
doesn't this statement need an await?

Reply:
It did, but I updated it to use the migrations.